lib/resty/core/regex.lua

```diff
@@ -153,6 +154,14 @@ local function get_max_regex_cache_size()
     return max_regex_cache_size
 end
 
+function _M.regex_cache_is_empty()
+    if not regex_match_cache then
+        return true
+    else
+        return next(regex_match_cache.hasht) == nil
```

Comment:
@alubbe Okay, I think it's better just to set the flag in our own wrapper as `lrucache_set()`. Right now `lrucache_set()` is pointed directly to `lrucache.set` and we can turn it into a wrapper around `lrucache.set`.